stream_interfaces.go

```diff
@@ -20,56 +20,99 @@ package grpc
 
 // ServerStreamingClient represents the client side of a server-streaming (one
 // request, many responses) RPC. It is generic over the type of the response
-// message. It is used in generated code.
+// message. If an error occurs on the stream, it will be returned as an instance
+// of the status package. Refer to the status package documentation for more
+// details. It is used in generated code.
 type ServerStreamingClient[Res any] interface {
+	// Recv receives the next message from the server. The client can repeatedly
+	// call Recv to read messages from the server-to-client response stream. Recv
+	// returns (nil, io.EOF) once the server-to-client stream is completely read.
 	Recv() (*Res, error)
 	ClientStream
 }
 
 // ServerStreamingServer represents the server side of a server-streaming (one
 // request, many responses) RPC. It is generic over the type of the response
-// message. It is used in generated code.
+// message. If an error occurs on the stream, it will be returned as an instance
+// of the status package. Refer to the status package documentation for more
+// details. End-of-stream for the server-to-client stream is indicated by the
+// return of the handler method. It is used in generated code.
 type ServerStreamingServer[Res any] interface {
+	// Send can send a stream of messages to the client. It may be called multiple
```

Comment:
`Server handler may call Send multiple to time send multiple messages to the client`